client/src/api/merger/merger.api.ts

```diff
@@ -35,6 +36,12 @@ export const uploadFile = async (file: RcFile, parser: string) => {
     .then((response) => response.data);
 };
 
+export const uploadFileToAnalyse = (file?:any) =>{
+  return axios
+    .get<UploadFileToAnalyseResponse>("/analyse/")
+    .then((response) => JSON.parse(response.data.json));
+}
+
```

Comment:
We should avoid having to download and then upload this file.

Better to create an endpoint `/transactions/stats` which will perform the necessary calculations, since all the data is already in the database